test/e2e/cluster_test.go

```diff
@@ -234,8 +235,21 @@ func TestClusterDeleteDenied(t *testing.T) {
 }
 
 func TestClusterDelete(t *testing.T) {
+	// Running this test requires knowing which context to use.
+	// For Argo CD CI, the k3s context is default, but this can
+	// vary if running locally. Use ARGOCD_E2E_CONTEXT_NAME to
+	// run this test locally using a specific context name (e.g. minikube).
+	var ctxName string
+	if os.Getenv("ARGOCD_E2E_K3S") == "true" {
+		ctxName = "default"
+	} else if os.Getenv("ARGOCD_E2E_CONTEXT_NAME") != "" {
+		ctxName = os.Getenv("ARGOCD_E2E_CONTEXT_NAME")
+	} else {
+		t.Skip()
```

Comment:
shouldn't be ctxName=default and if env variable ARGOCD_E2E_CONTEXT_NAME name is set overwrite ctxname and no need to add skip

Reply:
I had thought about this and was concerned with breaking local development/downstream tests. So I wanted to be explicit about the context to be used. That's probably already happened unfortunately though, so maybe it's fine to just assume default context for now. 

An alternative would be to assume the current context used is always what ctxName should be. I am beginning to think this is actually the path forward here.

Reply:
I went ahead and went with the approach of just using the current context instead of switching. Thanks for your review and let me know your thoughts.